xml/admin_ceph_gateway.xml

```diff
@@ -1927,7 +1927,7 @@ number_of_objects_expected_in_a_bucket / 100000
    <para>
     Be aware that the maximum number of shards is 7877.
    </para>
-   <sect3>
+<!--   <sect3 xml:id="rgw-simple-config">
```

Comment:
@theanalyst - what about this section? Remove?

Reply:
I don't think we need to remove as such. The params themselves are the same, just the delivery is different. just that there are 2 ways of configuring RGW, using `ceph config` and of course updating the ceph.conf files, though former is more preferred